Makefile

```diff
@@ -370,6 +370,7 @@ version: mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mod.yaml m
 docs: utility mods version
 	@mono --debug OpenRA.Utility.exe all --docs > DOCUMENTATION.md
```

Comment:
It is not really an application programming interface. I will rename it to match the wiki name instead.